setup.cfg

```diff
@@ -79,5 +81,19 @@ install_requires =
 console_scripts =
   ansible-lint = ansiblelint.__main__:main
 
+[options.extras_require]
+# Anyone wanting the full ansible (ACD) should mention it as a standalone
+# dependency as our extras bring the slim versions.
+
+# current will become to ansible-core>=2.11 once is released
+current =
+  ansible-base>=2.10
+devel =
+  ansible-core @ git+https://github.com/ansible/ansible.git
+ansible210 =
+  ansible-base>=2.10
```

Comment:
I really don't think `ansible210` should refer to ansible-*base*. It should refer to Ansible, or it should be called `ansiblebase210`.

Reply:
I reworked the extras we defined to avoid generating too many of them. Latest version does expose only 3 extras and users that do not want to use them are welcomed to BYOD (dependency). I also updated the install readme in order to clarify installation options.